doc/vxlan/data_plane_VXLAN_HLD.md

```diff
@@ -0,0 +1,510 @@
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_DATAPLANE_VTEP_TABLE
+    deactivate APP_DB
+    deactivate vxlanmgr
+
+ 
+    A->>vxlanmgr: Add VXLAN_REMOTE_VTEP_VNI_TABLE
+    activate vxlanmgr
+    vxlanmgr->>APP_DB: Add VXLAN_DATAPLANE_REMOTE_VNI_TABLE
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_DATAPLANE_REMOTE_VNI_TABLE
+    activate vxlanorch
+    vxlanorch->>ASIC_DB: sai_tunnel_api->create_tunnel,<br>sai_tunnel_api->create_tunnel_term,
+    vxlanorch->>portorch: addTunnel
+    portorch->>ASIC_DB: sai_bridge_api->create_bridge_port with learning enabled
+    vxlanorch->>portorch: addVlanMember
+    portorch->>ASIC_DB: sai_vlan_api->create_vlan_member
+    deactivate APP_DB
+    deactivate vxlanmgr
+    deactivate vxlanorch
+
+```
+
+##### Figure 3. Dataplane tunnel removal
+```mermaid
+%%{init: {'theme':'base', 'themeVariables':{'primaryColor': '#FFE068', 'fontFamily': 'trebuchet ", 'fontSize': '30px', "signalColor": "brown"}}}%%
+sequenceDiagram
+    participant A as CLI script
+    participant vxlanmgr
+    participant APP_DB as APP DB
+    participant vxlanorch
+    participant portorch
+    participant ASIC_DB as ASIC DB
+
+    A->>vxlanmgr: Del VXLAN_REMOTE_VTEP_VNI_TABLE
+    activate vxlanmgr
+    vxlanmgr->>APP_DB: Del VXLAN_DATAPLANE_REMOTE_VNI_TABLE
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_DATAPLANE_REMOTE_VNI_TABLE
+    activate vxlanorch
+    vxlanorch->>portorch: removeVlanMember
+    portorch->>ASIC_DB: sai_vlan_api->remove_vlan_member
+    vxlanorch->>portorch: delTunnel
+    portorch->>ASIC_DB: sai_bridge_api->remove_bridge_port
+    vxlanorch->>ASIC_DB: sai_tunnel_api->remove_tunnel_term_table_entry,<br>sai_tunnel_api->remove_tunnel
+    deactivate APP_DB
+    deactivate vxlanmgr
+    deactivate vxlanorch
+
+    A->>vxlanmgr: Del VXLAN_REMOTE_VTEP_TABLE
+    activate vxlanmgr
+    Note right of vxlanmgr: on last entry
+    vxlanmgr->>APP_DB: Del VXLAN_DATAPLANE_VTEP_TABLE
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_DATAPLANE_VTEP_TABLE
+    deactivate APP_DB
+    deactivate vxlanmgr
+
+    A->>vxlanmgr: Del VXLAN_TUNNEL_MAP
+    activate vxlanmgr
+    vxlanmgr->>APP_DB: VXLAN_TUNNEL_MAP
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_TUNNEL_MAP
+    activate vxlanorch
+    vxlanorch->>ASIC_DB: sai_tunnel_api->remove_tunnel,<br>sai_tunnel_api->remote_tunnel_map
+    deactivate vxlanmgr
+    deactivate APP_DB
+    deactivate vxlanorch
+
+    A->>vxlanmgr: Del VXLAN_TUNNEL_TABLE
+    activate vxlanmgr
+    vxlanmgr->>APP_DB: VXLAN_TUNNEL_TABLE
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_TUNNEL_TABLE
+    deactivate vxlanmgr
+    deactivate APP_DB
+```
+
+
+#### 4.3.2 P2MP Tunnel Creation
+    In the current implementation P2MP tunnel creation flow exist with the exception of a bridgeport not created for P2MP tunnel. To support using P2MP tunnel for L2 purposes a bridge port is created for the P2MP tunnel object.
+   
+    * Can/should it be supported in case of data plane tunnel ?
+
```

Comment:
please explain the npon creating brg port , i think i does create on tunnel termination 
if we support p2mp it means that user must configure every mac statically so i dont think this is needed

Reply:
Brigdeport explanation is taken from old HLD by mistake.
I rephrased the section, please changed text.